cmd/vic-machine/configure/configure.go

```diff
@@ -111,6 +113,12 @@ func (c *Configure) processParams() error {
 		return err
 	}
 
+	var err error
+	usingDHCP := ip.IsUnspecifiedIP(c.Data.ClientNetwork.IP.IP) || ip.IsUnspecifiedIP(c.Data.PublicNetwork.IP.IP) || ip.IsUnspecifiedIP(c.Data.ManagementNetwork.IP.IP)
```

Comment:
this method is to test value set in data structure, so it should be fine to have the method in data.go